beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestWithUsagePlugin.java

```diff
@@ -202,6 +202,25 @@ public TestUsagePluginApi() {
             return result;
         }
 
+        @Override
+        public List<RawUsageRecord> getUsageForSubscription(final UUID subscriptionId, final LocalDate startDate, final LocalDate endDate, final TenantContext tenantContext) {
+            final List<RawUsageRecord> result = new LinkedList<>();
```

Comment:
Nit: you could have called `getUsageForAccount` instead, and do the filtering afterwards to avoid duplicating the date logic.

Reply:
ack - will change the code.